password_generator_v2.0.py

```diff
@@ -0,0 +1,75 @@
+import secrets
+import string
+
+
+def char_count():
+    while True:
+        PWD_LENGTH = input("Enter a desired character count: ")
+        try:
+            PWD_LENGTH = int(PWD_LENGTH)
+        except ValueError:
+            print(
+                "Please enter integer number as a"
+                "desired password character count: "
+            )
+            continue
+        if PWD_LENGTH < 1:
+            print("Password character count must be greater than zero: ")
+            continue
+        return PWD_LENGTH
+
+
+def exclusion_choice(m, chars):
+    while True:
+        m = input(f"Exclude characters like '{chars}' ? Enter yes/no: ")
+        try:
+            m = str(m)
+        except ValueError:
+            print("Please enter yes/no: ")
+            continue
+        if m == 'yes':
+            return True
+        elif m == 'no':
+            return False
+        else:
+            continue
+
+
+def pwd_generator(PWD_LENGTH):
```

Comment:
Заглавными пишут только константы. В данном случае это параметр

```suggestion
def pwd_generator(pwd_length):
```